src/main/java/stelnet/commodity/market/price/PriceFactory.java

```diff
@@ -1,11 +1,11 @@
-package stelnet.commodity.data;
+package stelnet.commodity.market.price;
 
-import stelnet.commodity.CommodityBoard.CommodityTab;
+import stelnet.commodity.CommodityTab;
 
 public class PriceFactory {
 
     public Price get(String commodityId, CommodityTab commodityTab) {
-        Price price = new DummyPrice();
+        Price price = null;
```

Comment:
DummyPrice was to avoid NPE without having to check for nulls.

Reply:
Today yes, tomorrow (when another contributor touches it) not so much. It's defensive programming, but I'll leave it as is.

Reply:
Theoretically we could remove this class all-together and ask for the price from MarketWrapperApi class.

Reply:
Seems like we are no longer using Price object in CommodityIntel either, so we could actually drop this.

Reply:
I removed it. I noticed now that the "price" instance variable in CommodityIntel is also questionable. There is this interesting thing there, which I am unsure what it does exactly:

````
    @Override
    public boolean isEnding() {
        return Math.abs(price - priceProvider.getPrice(market)) > 1;
    }
 ````
 is now:
 ````
     @Override
    public boolean isEnding() {
        return Math.abs(price - market.getPriceAmount()) > 1;
    }
 ````
It seems like this always returns 0 (the substraction). Because it is essentially same as : ``market.getPriceAmount - market.getPriceAmount()`` - I did not change the logic here I think.

Reply:
In master CommodityIntel is created like this:

````
            Price price = priceFactory.get(commodityId, commodityTab);
            intel = new CommodityIntel(action, commodity, market, this, price);
````
So the logic of it is the same, it always substracts itself from itself. Or am I misunderstanding?

Reply:
`price` is price at snapshot (when you clicked to track this as small intel)
`market.getPriceAmount()` is current price, which will start to differ the more time passes (convoys arriving, month ticking)

Reply:
Removed PriceFactory. Kept the rest.